packages/app-admin-ui/index.js

```diff
@@ -19,6 +19,8 @@ class AdminUIApp {
     enableDefaultRoute = false,
     isAccessAllowed = () => true,
     schemaName = 'public',
+    enableDevFeatures = false,
+    buildMode = process.env.NODE_ENV === 'production' ? 'production' : 'development',
```

Comment:
This option isn't necessary, it's already passed into the `build` and `prepareMiddleware` methods as the `dev` argument.

Reply:
It's used by the `getWebpackConfig` function. Are you saying that after the `getWebpackConfig` is called this value is then overwritten?

If so what do you want to do? Strip this out and then set 'production' as the default in `getWebpackConfig`?

Reply:
Ok so it looks like build should always be production and createDevMiddleware always development. Love it! Deleting code :)